.github/workflows/main.yml

```diff
@@ -52,7 +52,7 @@ jobs:
             galsim \
             ngmix
           
-          python -m pip install tensorflow==2.5.0 tensorflow_addons==0.13.0 tensorflow_probability==0.13.0
+          python -m pip install tensorflow==2.5.0 tensorflow_addons==0.13.0 tensorflow_probability==0.13.0 numdifftools
```

Comment:
do we need numdifftools here?

Reply:
It's not used in this test, but in the other. I don't think there is a problem in being here.

Reply:
we shouldnd't add  dependencies that are not neeeded